raft.go

```diff
@@ -197,6 +186,47 @@ func (r *Raft) runFollower() {
 	}
 }
 
+// liveBootstrap attempts to seed an initial configuration for the cluster. See
+// the Raft object's member BootstrapCluster for more details. This must only be
+// called on the main thread, and only makes sense in the follower state.
+func (r *Raft) liveBootstrap(configuration Configuration) error {
+	// Sanity check the Raft peer configuration.
```

Comment:
A lot of this code is the same as non-live bootstrap. Is it better to call that here?